services/readXls.ts

```diff
@@ -86,9 +86,6 @@ function getContributionSubType(orestarSubType: string): ContributionSubType {
     'In-Kind Contribution': ContributionSubType.INKIND_CONTRIBUTION,
     'In-Kind/Forgiven Personal Expenditures': ContributionSubType.INKIND_FORGIVEN_PERSONAL,
     'In-Kind/Forgiven Account Payable': ContributionSubType.INKIND_FORGIVEN_ACCOUNT,
-    // 'Loan Received (Non-Exempt)': ContributionSubType.INKIND_CONTRIBUTION,
-    // 'Pledge of Loan': ContributionSubType.INKIND_CONTRIBUTION,
-    // 'Pledge of In-Kind': ContributionSubType.INKIND_CONTRIBUTION
```

Comment:
we should account for these or at least keep documented that they are not accounted for.

Reply:
Documenting that we're not accounting for them is fine with me!

Reply:
Ok, revised plan. I say the we should keep the Loan Received (Non-Exempt) as OTHER, and not include the pledges.